### DISCUSSION TITLE
使rolling release 20231218135354版翻译的缓存文件，可以在20240304213912版中使用吗？

### DISCUSSION THREAD

**Discussion by GoTop** [bookfere/Ebook-Translator-Calibre-Plugin#225] [Q&A]
20231218135354版的没有检测原文和译文段落数的功能

20240304213912版有的段落会被标记为黄色

**Comment by bookfere** [comment]
插件对旧版本的数据做了兼容，应该是可以使用的，但是有可能存在问题。

另，非功能请求和插件错误的内容，请以用合适分类发布到 [Discussions](https://github.com/bookfere/Ebook-Translator-Calibre-Plugin/discussions) 上。

**Comment by GoTop** [comment]
> 插件对旧版本的数据做了兼容，应该是可以使用的，但是有可能存在问题。
> 
> 另，非功能请求和插件错误的内容，请以用合适分类发布到 [Discussions](https://github.com/bookfere/Ebook-Translator-Calibre-Plugin/discussions) 上。

请教不同的版本对于合并段落的划分有什么不同？

**Reply by bookfere** [reply]
之前用的是 `{{id_123}}` 这种标记分割段落，现在用的是两个换行符 `\n\n` 分割段落。
